bank/README.md

```diff
@@ -0,0 +1,62 @@
+# Bank Module
+
+## Table of Contents
+
+* [Overview](#overview)
+* [How does it work](#how-does-it-work)
+* [Query commands](#query-commands)
+* [Transaction commands](#transaction-commands)
+
+## Overview
+
+The bank module is responsible for handling multi-asset coin transfers between accounts. It exposes several interfaces with varying capabilities for secure interaction with other modules which must alter user balances.
+
+## How does it work
+
+```
+type MsgSend struct {
+	FromAddress types.HeimdallAddress `json:"from_address"`
+	ToAddress   types.HeimdallAddress `json:"to_address"`
+	Amount      sdk.Coins             `json:"amount"`
+}
+```
+
+[Handler](handler.go) for this transaction validates whether send is enabled or not
+
+Once the event is validated by the Handler, It will go send coins a particular amount of coins to the sender
+
+## Query commands
+
+One can run the following query commands from the bank module :
+
+* `balance` - Query for bank balance of an address.
+
+### CLI commands
```

Comment:
Maybe this structure does not comply with what we have defined [here](https://0xpolygon.slack.com/archives/C03PD53G8L9/p1697523603092529?thread_ts=1697449775.524569&cid=C03PD53G8L9)